docs/topics/contracts.rst

```diff
@@ -46,6 +46,14 @@ This callback is tested using three built-in contracts:
 
     @cb_kwargs {"arg1": "value1", "arg2": "value2", ...}
 
+.. class:: MetadataContract
```

Comment:
Above in this file we have:

https://github.com/scrapy/scrapy/blob/6ce0342beb1a5b588f353e52fe03d5e0ec84d938/docs/topics/contracts.rst?plain=1#L29

I'm not sure if this statement is correct since we just just are listing the default contracts and `cb_kwargs` is not used in the snippet:

https://github.com/scrapy/scrapy/blob/6ce0342beb1a5b588f353e52fe03d5e0ec84d938/docs/topics/contracts.rst?plain=1#L23-L26

I followed the pattern and added the `MetadataContract` to the list.

Reply:
I would reword that to:

> You can use the following contracts:

Also, I think we should replace amazon.com with example.com.

Reply:
Done, thanks.